### PR TITLE
test

### PR DESCRIPTION
**Pull request by DorianBajorek**

### CODE REVIEW

File: src/views/codeEditor/index.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ export default function CodeEditor() {
     const [code, setCode] = useState<string>();
 
     const sendCode = () => {
-        console.log(code);
+        console.log(code+"test");
```
Comment (Lukasz-gro) [inline src/views/codeEditor/index.tsx:9]: Is this necessary?

Comment (DorianBajorek) [reply src/views/codeEditor/index.tsx:9]: yes its very important

File: src/views/about/About.tsx [hunk 2]
```diff
@@ -1,19 +1,47 @@
...
-    theme='tomorrow'
-    code={code}
-    lineNumber
-    readOnly={false}
-    clipboard={true}
-    changeCode={(code: any) => {
-      console.log(code);
-    }}
-/>;
+  const [language, setLanguage] = useState<Language>(Language.CPP);
```
Comment (Lukasz-gro) [inline src/views/about/About.tsx]: Add enter after this line

File: src/views/about/About.tsx [hunk 3]
```diff
@@ -1,19 +1,69 @@
...
+  MEDIUM = "medium", 
+  LARGE = "large"
+}
+
+const mapOfLanguages = new Map<Language, string>()
+    .set(Language.CPP, "cpp")
+    .set(Language.JAVA, "java")
+    .set(Language.PYTHON, "python");
+
+const mapOfCodeSize = new Map<CodeSize, string>()
```
Comment (Lukasz-gro) [inline src/views/about/About.tsx]: This map is not necessary as we use CodeSize has the same values

Comment (DorianBajorek) [reply src/views/about/About.tsx]: You are right!

**Review by Lukasz-gro — APPROVED** [review]

**Pull request merged**